engine/src/main/java/org/hibernate/validator/internal/xml/ConstrainedExecutableBuilder.java

```diff
@@ -211,7 +213,7 @@ private ConstrainedExecutable parseExecutableType(String defaultPackage,
 		Set<MetaConstraint<?>> returnValueTypeArgumentConstraints = new HashSet<>();
 		List<CascadingTypeParameter> cascadingTypeParameters = new ArrayList<>();
 		Map<Class<?>, Class<?>> groupConversions = new HashMap<>();
```

Comment:
Do we still need this var? (Probably yes, just looking at the GH diff, so lacking the full context).